Dockerfile

```diff
@@ -2,7 +2,7 @@ ARG GRAFANA_VERSION="latest"
 
 FROM grafana/grafana:${GRAFANA_VERSION}
 
-ARG OPENNMS_HELM_VERSION="bleeding"
+ARG OPENNMS_GRAFANA-PLUGIN_VERSION="bleeding"
```

Comment:
Probably should be `OPENNMS_GRAFANA_PLUGIN_VERSION` (underscore instead of hyphen)